stock_batch_picking/i18n/stock_batch_picking.pot

```diff
@@ -0,0 +1,395 @@
+# Translation of Odoo Server.
```

Comment:
If the guideline changed after https://github.com/OCA/maintainer-tools/issues/97 could add it in our guidelines?